app/javascript/react/components/StationInfoTile.js

```diff
@@ -17,25 +17,23 @@ const StationInfoTile = props => {
     .then(response => response.json())
     .then(fetchedData => {
       setStations(fetchedData)
-      // setLoadComplete(true)
     })
   }, [])
```

Comment:
currently you make this fetch twice, and so possibly have two sources of truth in your status and info tiles. 

I would move your fetch to the index container, and then pass down the fetched data from there
Im not 100% what info you are gathering from this fetch, but you might even be served by merging the status and info tiles into one if status data and info data are about the same station